lib/valkyrie/persistence/fedora/list_node.rb

```diff
@@ -59,24 +58,22 @@ def prev
     def to_graph
       return RDF::Graph.new if target_id.blank?
       g = Resource.new(rdf_subject)
-      g.proxy_for = target_uri
+      g.proxy_for = target
       g.proxy_in = proxy_in.try(:uri)
       g.next = self.next.try(:rdf_subject)
       g.prev = prev.try(:rdf_subject)
       g.graph
     end
 
-    # Resolves the URI for the value of the list expression
```

Comment:
Can you keep the comment on this method?

Reply:
Does it still necessarily resolve to a URI?

Reply:
I'm convinced that a comment doesn't really help here, since it would be more complicated to explain than to just read the code.